warrior/Framework/Utils/data_Utils.py

```diff
@@ -1490,29 +1490,6 @@ def verify_inorder_cmd_response(match_list, verify_list, system, command,
     return status
 
 
-def get_cse_script_args_string(datafile, system_name):
```

Comment:
@satyakrish 
This PR is to remove CSE script execution related options from WF.
This util method(get_cse_script_args_string) is specific to CSE script execution and it's usage in non-native warrior keywords is very remote. Should we keep it for backward compatibility?

Reply:
Spoke to Sathya, we agreed to remove this util method.